packages/features/bookings/components/FiltersContainer.tsx

```diff
@@ -1,51 +1,135 @@
-import type { ReactNode } from "react";
-import { Fragment } from "react";
+import { useState } from "react";
+import { shallow } from "zustand/shallow";
 
+import { useFilterQuery } from "@calcom/features/bookings/lib/useFilterQuery";
+import {
+  TeamsFilter,
+  FilterCheckboxFieldsContainer,
+  FilterCheckboxField,
+} from "@calcom/features/filters/components/TeamsFilter";
+import { WEBAPP_URL } from "@calcom/lib/constants";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { trpc } from "@calcom/trpc/react";
+import {
+  Dropdown,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownItem,
+  DropdownMenuTrigger,
+  AnimatedPopover,
+  Avatar,
+  FilterSearchField,
+  Tooltip,
+} from "@calcom/ui";
+import { Plus } from "@calcom/ui/components/icon";
+
+import { useBookingMultiFilterStore } from "../BookingMultiFiltersStore";
 import { EventTypeFilter } from "./EventTypeFilter";
-import { PeopleFilter } from "./PeopleFilter";
-import { TeamsMemberFilter } from "./TeamFilter";
 
-type FilterTypes = "teams" | "people" | "eventType";
+const PeopleFilter = () => {
+  const { t } = useLocale();
+  const { data: query, pushItemToKey, removeItemByKeyAndValue, removeByKey } = useFilterQuery();
+  const [searchText, setSearchText] = useState("");
 
-type Filter = {
-  name: FilterTypes;
-  controllingQueryParams?: string[]; // this is what the filter controls - but also we show the filter if any of these query params are present
-  component: ReactNode;
-  showByDefault?: boolean;
-};
+  const members = trpc.viewer.teams.listMembers.useQuery({});
+
+  const filteredMembers = members?.data
+    ?.filter((member) => member.accepted)
```

Comment:
nit: It's already fetching accepted team members only. So it might not be necessary